tests/python_package_test/test_sklearn.py

```diff
@@ -112,21 +112,33 @@ def test_multiclass():
     assert gbm.evals_result_['valid_0']['multi_logloss'][gbm.best_iteration_ - 1] == pytest.approx(ret)
 
 
-def test_lambdarank():
+def lambdarank_test_runner(lambdarank_unbiased=False, **kwargs):
```

Comment:
thanks very much for adding tests!

I think we could have higher confidence that these changes are backwards-compatible if the existing tests were not touched at all. That sort of acts as a proxy for users' existing code.

I appreciate the effort at reducing duplicated code here, but could you please instead just add new tests for this feature and leave the others untouched? It's ok if that means copying and pasting the contents of `test_lambdarank`.